systemFlake.nix

```diff
@@ -86,24 +86,51 @@ let
     let
       selectedNixpkgs = getNixpkgs host;
       host = evalHostArgs (mergeAny hostDefaults host');
+      patchedChannel = selectedNixpkgs.path;
+      # Use lib from patched nixpkgs
+      lib = selectedNixpkgs.lib;
+      # Use nixos modules from patched nixpkgs
+      baseModules = import (patchedChannel + "/nixos/modules/module-list.nix");
+      # Override `modulesPath` because otherwise imports from there will not use patched nixpkgs
+      specialArgs = { modulesPath = builtins.toString (patchedChannel + "/nixos/modules"); } // host.specialArgs;
+      # The only way to find out if a host has `nixpkgs.config` set to
+      # the non-default value is by evalling most of the config.
```

Comment:
I'm quite happy with it. I'll take a closer look into those other two cases at... at... at some point :D
But yeah, LGTM

Reply:
We definitely need to find somebody who knows how to implement `inputs.myflake.patches = [];` :smile: